tests/test-cases/Group6-VIC-Machine/6-13-TLS.robot

```diff
@@ -18,6 +18,20 @@ Resource  ../../resources/Util.robot
 Test Teardown  Run Keyword If Test Failed  Cleanup VIC Appliance On Test Server
 Test Timeout  20 minutes
 
+*** Keywords ***
+Check that requests with invalid host field are rejected
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -vvv -H"Host: please.ddos" %{DOCKER_HOST}/_ping
+    Should Contain  ${output}  invalid host header
+    Should Contain  ${output}  500 Internal Server Error
+    Should Contain  ${output}  to requested host please.ddos
+    Should Not Contain  ${output}  200 OK
+
+Check that normal requests are accepted
+    ${rc}  ${output}=  Run And Return Rc And Output  curl -vvv %{DOCKER_HOST}/_ping
```

Comment:
Can you confirm that curl adds DOCKER_HOST as the host header if we're not explicit?

Reply:
I know that's curl's behavior, yeah:
```
 ❯ curl -vvv  10.193.63.157:2375/_ping                                                                                           [16:07:07]
*   Trying 10.193.63.157...
* TCP_NODELAY set
* Connected to 10.193.63.157 (10.193.63.157) port 2375 (#0)
> GET /_ping HTTP/1.1
> Host: 10.193.63.157:2375
> User-Agent: curl/7.59.0
> Accept: */*
> 
< HTTP/1.1 200 OK
< Api-Version: 1.25
< Server: Docker/1.25 (linux)
< Date: Wed, 11 Apr 2018 21:07:50 GMT
< Content-Length: 2
< Content-Type: text/plain; charset=utf-8
< 
* Connection #0 to host 10.193.63.157 left intact
OK%                                                                                                                                         
```

Reply:
Unless you're referring to a confirmation in the test, another `Should Contain` @hickeng ?

Reply:
I'm ok with it as is. Might be nice to have one without the port in there to validate that path.

Reply:
Added the might-be-nice case too :)